### ISSUE TITLE
Laravel 5.4 support

### ISSUE THREAD

**Issue by ingageco** [liebig/cron#88]
Upon composer update, only received this error: 

Call to undefined method Illuminate\Foundation\Application::share() in /var/www/vendor/liebig/cron/src/Liebig/Cron/Laravel5ServiceProvider.php on line 71

Laravel upgrade guide documents the removal of the share() method.

**Comment by liebig** [comment]
Yep, the share method was removed.

`The share method has been removed from the container. This was a legacy method that has not been documented in several years. If you are using this method, you should begin using the  singleton method instead:`
``` php
$container->singleton('foo', function () {
    return 'foo';
});
```

So the lines 71, 79, 83, 87 of the Laravel5ServiceProvider.php have to be changed. A pull request would be awesome.

**Comment by liebig** [comment]
Should be fixed with commit 	d56bfbc. Please update to head, test and give a short reply. Thank you.

**Comment by liebig** [comment]
Change is tested and released.
